mongodb/mongodb.php

```diff
@@ -1159,7 +1159,7 @@ class Regex implements Type
              * @param string $pattern
              * @param string $flags
```

Comment:
please add [optional] 
string $flags -> string #flags [optional]

Reply:
I added the [optional] flat, but I didn't see that anywhere else in the file...

Reply:
We use it in our stubs for optional parameters